src/main/java/com/sonyericsson/hudson/plugins/gerrit/trigger/gerritnotifier/ParameterExpander.java

```diff
@@ -365,7 +370,7 @@ protected int getVerifiedValue(Result res, GerritTrigger trigger) {
      * @return the lowest verified value.
      */
     public int getMinimumVerifiedValue(MemoryImprint memoryImprint, boolean onlyBuilt) {
```

Comment:
If possible, to be more correct I think this method should return ```null``` if no minimum value is found at all. It would be more correct than returning ```Integer.MAX_VALUE``` that is definitely not a number that was found.

The method should also be annotated with ```CheckForNull``` to help findbugs, and the javadoc should be updated to note the new behaviour.